imports/plugins/core/dashboard/client/components/StorefrontUrls.js

```diff
@@ -20,6 +20,11 @@ import { i18next } from "/client/api";
 import withPrimaryShopId from "/imports/plugins/core/graphql/lib/hocs/withPrimaryShopId";
 import withShop from "/imports/plugins/core/graphql/lib/hocs/withShop";
 
+const CardContainer = styled(Card)`
```

Comment:
Let's get rid of the styled-components use altogether. `Card` and `CardContent` from MUI already have the appropriate styles.

PaddedField can be done with `Box`.

And `RightAlignedGrid` and where it's used in `CardActions` could be simplified. 

This
```jsx
                <CardActions>
                  <Grid container alignItems="center" justify="flex-end">
                    <RightAlignedGrid item xs={12}>
                      <Button color="primary" variant="contained" onClick={this.handleSubmitForm}>
                        {i18next.t("app.save")}
                      </Button>
                    </RightAlignedGrid>
                  </Grid>
                </CardActions>
```

Could be, this instead.

```jsx
                <CardActions>
                  <Box textAlign="right">
                      <Button color="primary" variant="contained" onClick={this.handleSubmitForm}>
                        {i18next.t("app.save")}
                      </Button>
                </CardActions>
```

Reply:
refactored